docs/Tutorials/Developer-Quickstart.md

```diff
@@ -65,3 +66,99 @@ the tutorial.
+- A list of nodes to connect to; this is done by specifying [bootnodes], or by providing a list of [static nodes].
+- A node key pair and optionally an account. If the running network is using permissions, then you need
+  to add the new node's enode details to the [permissions file] used by existing nodes, or update
+  the onchain permissioning contract.
+  The following steps describe the process to add a new node to the Quorum Dev Quickstart.
+
+### 1. Create the node key files
+
+Create a node key pair and account for a new node, by running the following script:
+
+```bash
+cd ./extra
+npm install
+node generate_node_keys.js --password "Password"
+```
+
+!!! note
+
+    The `--password` parameter is optional.
+
+### 2. Create new node directory
+
+In the [`config/nodes`](https://github.com/ConsenSys/quorum-dev-quickstart/tree/master/files/common/config/nodes)
+directory, create a subdirectory for the new node (for example, `newnode`), and move the
+`nodekey`, `nodekey.pub`, `address` and `accountkey` files from the previous step into this directory.
+
+### 3. Update docker-compose
+
+Add an entry for the new node into the docker-compose file:
+
+```yaml
+  newnode:
+    <<: *besu-def
+    container_name: newnode
+    volumes:
+      - public-keys:/opt/besu/public-keys/
+      - ./config/besu/:/config
+      - ./config/nodes/newnode:/opt/besu/keys
+      - ./logs/besu:/tmp/besu
+    depends_on:
+      - validator1
+    networks:
+      quorum-dev-quickstart:
+        ipv4_address: 172.16.239.41
+```
+
+!!! important
+
+    Select an IP address and port map that aren't being used for the other containers. Additionally mount the newly created
+    folder `./config/nodes/newnode` to the `/opt/besu/keys` directory of the new node, as seen in the example above.
+
+### 4. Update Prometheus configuration
+
+Update `prometheus.yml` under [`./config/prometheus/`](https://github.com/ConsenSys/quorum-dev-quickstart/tree/master/files/besu/config/prometheus) to enable metrics gathering and to show up under Grafana.
+Insert the following under `scrape_configs` section in the file. Make sure to change the `job_name` and `targets` appropriately if you have changed them:
+
+```yaml
+- job_name: newnode
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+    - targets: [newnode:9545]
+```
+
+### 5. Update files with the enode address
+
+Add the new node's enode address to the [static nodes] file and [permissions file].
+The enode uses the format `enode://pubkey@ip_address:30303`.
+If the `nodekey.pub` is `4540ea...9c1d78` and the IP address is `172.16.239.41`, then the enode
+address would be `"enode://4540ea...9c1d78@172.16.239.41:30303"`,
+which must be added to both files.
+
+As an alternative, it is also possible to call the Besu API method [`perm_addNodesToAllowlist`](https://besu.hyperledger.org/en/latest/Reference/API-Methods/#perm_addnodestoallowlist) to add the new node to the [allow list].
```

Comment:
Suggest reword to `For the permissioned allow list, you can alternatively call the API method ..... to add the new node`